rspace/src/main/scala/coop/rchain/rspace/merger/package.scala

```diff
@@ -48,7 +48,11 @@ package object merger {
       }
     }
     // each rejection option is defined by decision not to reject a key in rejection map
-    conflictMap.keySet.map(k => process(conflictMap(k), Set.empty, reject = true))
+    conflictMap
+    // only keys that have conflicts associated should be examined
+      .filter { case (_, conflicts) => conflicts.nonEmpty }
+      .keySet
+      .map(k => process(conflictMap(k), Set.empty, reject = true))
```

Comment:
Without this filter rejection option to not reject anything present in output when there is a single branch that does not conflict with anything.

Reply:
Given this change, issue <https://github.com/rchain/rchain/issues/3349> is present again. Previously it was masked bu this bug.

Reply:
Why is this here and not in merging logic file?

Reply:
I'm not sure, Its sort of general operations on Maps. Do you think the whole package content should be in merging logic?

Reply:
Actually might be makes sense to put this particular fucntion to merging logic

Reply:
https://github.com/rchain/rchain/pull/3444/commits/3c058c47f0c17d3d843be0102ba2e6ff0170cdec